lib/builder/builder.js

```diff
@@ -438,8 +438,12 @@ export default class Builder {
       compilersOptions.push(serverConfig)
     }
 
-    // Alias plugins to their real path
-    this.plugins.forEach(p => {
+    // Check plugins exist then set alias to their real path
+    await Promise.all(this.plugins.map(async p => {
+      if (!await fsExtra.pathExists(p.src)) {
```

Comment:
Shouldn't we first resolve plugin's full path? Something like `~/plugins/my_plugin.js` would throw an error. right?